packages/eas-cli/src/credentials/android/utils/googleServiceAccountKey.ts

```diff
@@ -0,0 +1,12 @@
+import assert from 'assert';
+
+import { GoogleServiceAccountKey } from '../credentials';
+
+export function validateServiceAccountKey(keyJson: string): GoogleServiceAccountKey {
+  const jsonKeyObject = JSON.parse(keyJson);
+  assert(
+    'private_key' in jsonKeyObject && typeof jsonKeyObject['private_key'] === 'string',
+    'The provided JSON key file must contain the "private_key" field'
```

Comment:
some people are passing google-servic.json here, it would be nice to detect that case and print specific error message

Reply:
i've changed this to detect whether the user has uploaded a file path with `google-services` in it